docs/package.json

```diff
@@ -4,18 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "prettier": "prettier --write --config \"./.prettierrc\" \"**/*.{yml,json,html,css,js,md}\""
+    "reformat": "node ../scripts/reformat-docs-with-prettier.js"
   },
   "author": "",
-  "license": "ISC",
-  "devDependencies": {
-    "husky": "^1.3.1",
-    "prettier": "^1.16.4",
-    "pretty-quick": "^1.10.0"
-  },
-  "husky": {
```

Comment:
husky can not be isolated to one package/directory when the project is a mono repo. I've moved this to the top level of the repo.